docs/users-guide/13-embedding.md

```diff
@@ -0,0 +1,62 @@
+```
+SELECT count(*)
+FROM orders
+WHERE product_id = {{productID}}
+```
+you could specify a specific productID when embedding the question.
+
+#### Signed parameters
+In general, when embedding a chart or dashboard, the server of your embedding application will need to sign a request for that resource. 
+
+If you choose to sign a specific parameter value, that means the user can't modify that, nor is a filter widget displayed for that parameter. For example, if the "Publisher ID" is assigned a value and the request signed, that means the frontend client that renders that dashboard on behalf of a given logged-in user can only see information for that publisher ID. 
+
+### Enabling embedding
+To enable embedding, go to the Admin Panel and under Settings, go to the "Embedding in other applications" tab. From there, click "Enable." Here you will see a secret signing key you can use later to sign requests. If you ever need to invalidate that key and generate a new one, just click on "Regenerate Key". 
+![Enabling Embedding](images/embedding/01-enabling.png)
+
+You can also see all questions and dashboards that have been marked as "Embeddable" here, as well as revoke any questions or dashboards that should no longer be embeddable in other applications. 
+
+Once you've enabled the embedding feature on your Metabase instance, you should then go to the individual questions and dashboards you wish to embed to set them up for embedding.
+
+### Embedding Charts + Dashboards
+
+To mark a given question or dashboard, click on the sharing icon 
+
+![Share icon](images/embedding/02-share-icon.png)
+
+Then select "Embed this question in an application"
+
+![Enable sharing for a question](images/embedding/03-enable-question.png)
+
+Here you will see a preview of the question or dashboard as it will appear in your application, as well as a panel that whos you the code you will need to insert in your application. 
+
+![Preview](images/embedding/04-preview.png)
+
+Importantly, you will need to hit "Publish" when you first set up a  chart or dashboard for embedding and each time you change it. Also, any changes you make to the resource might require you to update the code in your own application to the latest code sample in the "Code Pane". 
+
+![Code samples for embedding](images/embedding/05-code.png)
+
+We provide code samples for common front end template languages as well as some common backend web frameworks and languages. You may also use these as starting points for writing your own versions in other platforms.
+
+
+### Embedding Charts + Dashboards with locked parameters
+If you wish to have a parameter locked down, to prevent your embedding application end users from seeing other users data, you can mark parameters as "Locked". Once a parameter is marked as Locked, it is not displayed as a filter widget, and must be set by the embedding application's server code. 
```

Comment:
A few things in this sentence: `If you wish to have a parameter locked down, to prevent your embedding application end users from seeing other users data, you can mark parameters as "Locked".`

Change to: `If you wish to have a parameter locked down to prevent your embedding application's end users from seeing other users' data, you can mark parameters as "Locked."`